networking/network_policy/about-network-policy.adoc

```diff
@@ -5,11 +5,666 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-As a cluster administrator, you can define network policies that restrict traffic to pods in your cluster.
+As a project administrator, you can define network policies that restrict traffic to or from pods in a namespace.
 
-include::modules/nw-networkpolicy-about.adoc[leveloffset=+1]
+[NOTE]
+====
+Any network policies defined by a cluster administrator can be overridden by a project administrator. Do not use network policies to define cluster-wide network traffic policy.
+====
 
-include::modules/nw-networkpolicy-optimize.adoc[leveloffset=+1]
+What follows is a discussion of how network traffic isolation works with network policy in {product-title}.
+
+[id="network-policy-concepts_{context}"]
+== Network policy concepts
+
+You can use a network policy to allow and deny network traffic within a particular namespace. The following concepts apply to network policy:
+
+* Network policy applies to pods in the namespace in which the policy is defined.
+* Network policy defines two types of rules:
+** Ingress rules that allow traffic to pods from other pods or IP address blocks.
+** Egress rules that allow traffic from pods to other pods or IP address blocks.
+* Network policy supports the TCP, UDP, and SCTP protocols.
+* Network policy supports specifying ports for supported protocols.
+
+A network policy consists of three parts:
```

Comment:
If anything I would remove the "Network Policy Concepts" header and add one here titled "The Network Policy Object"

Reply:
And I think this section should simply go under the sample Network Policy Object  (similar to what's done in [upstream docs](https://kubernetes.io/docs/concepts/services-networking/network-policies/#networkpolicy-resource))